js/models/listing/Metadata.js

```diff
@@ -17,19 +20,41 @@ export default class extends BaseModel {
       'PHYSICAL_GOOD',
       'DIGITAL_GOOD',
       'SERVICE',
+      'CRYPTOCURRENCY',
+    ];
+  }
+
+  get contractTypesVerbose() {
+    return this.contractTypes
+      .map((contractType) => (
+        {
+          code: contractType,
+          name: app.polyglot.t(`formats.${contractType}`),
+        }
+      ));
+  }
+
+  get formats() {
+    return [
+      'AUCTION',
```

Comment:
FYI, AUCTION wasn't being used. I recommended that @tyler remove it and add the new type. I was thinking it would replace 1 since it was never used, but I tried to keep my recommendations implementation-agnostic. OCD aside, I don't have a real problem with this.